### ISSUE TITLE
out of date packages

### ISSUE THREAD

**Issue by paulgibbs** [humanmade/react-wp-scripts#44]
When I try to use this project as per the readme instructions, the resulting package.json has alot of packages being flagged by `npm audit`:

```
react-wp-scripts paulgibbs$ npm audit fix
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.

up to date in 4.3s
fixed 0 of 325 vulnerabilities in 12576 scanned packages
  1 package update for 325 vulns involved breaking changes
  (use `npm audit fix --force` to install breaking changes; or refer to `npm audit` for steps to fix these manually)
```

`--force` resolves everything (there is a package that changes a major version number) but I am not sure if that would break anything.

**Comment by rmccue** [comment]
Activated dependabot to auto-fix this; see #46, #47, #48, #49

**Comment by rmccue** [comment]
Fixed ages ago.
